lib/litestack/litejobqueue.rb

```diff
@@ -32,7 +32,7 @@ class Litejobqueue < Litequeue
   # This can be particularly useful for long running, IO bound jobs. It is not recommended though for threaded environments, as it can result in creating many threads that may consudme a lot of memory. 
   DEFAULT_OPTIONS = {
     config_path: "./litejob.yml",
-    path: "./queue.db",
+    path: "./db/queue.db",
```

Comment:
Wouldn't defining the names based on the environment name be a better choice here, just like we do for the database?

Reply:
Yeah, but ultimately even this won't cut it. Most production environments will have a volume mounted outside of the Rails application. For example, the path might be `/data`. The databases should probably be put in a directory like `/data/production` and the databases named `/data/production/app.db`, `/data/production/queue.db`, etc.

Ultimately I want all of the configuration paths for databases in Litestack to support Procs since I want to make one database per tenant a possiblity.